core/node/events/snapshot.go

```diff
@@ -650,6 +663,11 @@ func update_Snapshot_Mls(
 		}
 		snapshot.Members[memberAddress].SignaturePublicKeys = append(snapshot.Members[memberAddress].SignaturePublicKeys, content.ExternalJoin.SignaturePublicKey)
 		return nil
+	case *MemberPayload_Mls_EpochSecrets_:
+		for _, secret := range content.EpochSecrets.Secrets {
+			snapshot.EpochSecrets[secret.Epoch] = secret.Secret
+		}
```

Comment:
right now if the secret is added twice, we're accepting the second one.. is that desired? normally first one wins